CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# XDG Desktop Portal - Contributing Guide
+
+Before developing features or fixing bugs, please make sure you are have done
+the following:
+
+- Your code is not on the *main* branch of your fork
+- The code has been tested
+- All commit messages are properly formatted and commits squashed where
+  appropriate
+- You have included updates to all appropriate documentation
+
+We use GitHub pull requests to review contributions. Please be kind and patient
+as code reviews can be long and minutious.
+
+## Development
+
+xdg-desktop-portal usually runs as a user session service, initialized before
+with the graphical session. xdg-desktop-portal initializes specific backends
+through D-Bus activation.
```

Comment:
Not really true is it? `xdg-desktop-portal` is D-Bus activated too thus starts as soon as anything requests it. In F38 it is effectively started immediately because gnome-software is autostarted and asks for `org.freedesktop.portal.Desktop`, but it doesn't have to be autostarted.

Reply:
You're right! Thank you